### PR TITLE
MOTECH-2182: bootstrap-fileUpload is deprecated

### PR DESCRIPTION
**Pull request by matkwiatkowski**

### CODE REVIEW

**Comment by kmadej** [comment]
After selecting a json file in message campaing module, the 'upload' button is unavailable (checked on firefox and chrome)

screenshot from 2016-03-17 11 28 35

**Comment by kmadej** [comment]
Do not remove the feature mentioned in https://github.com/motech/modules/commit/11b9c910c5401fba702fa558d7ef610c306618ee.
After updating file upload plugin, this update button should work as previously (details in mentioned commit or on JIRA https://applab.atlassian.net/browse/MOTECH-1495)

**Comment by matkwiatkowski** [comment]
I found that file upload event is not fired so I've added restriction to file input that accept only ".json" files and now submit button is shown only after selecting a file.

File: sms/src/main/resources/webapp/js/directives.js [hunk 1]
```diff
@@ -20,4 +21,18 @@
             }
         };
     });
+
+    // TODO: Can be removed after common file upload directive work again. (MOTECH-2265)
```
Comment (pgesek) [inline sms/src/main/resources/webapp/js/directives.js]: MOTECH-2265 is about the period modal not working after minification. What does this have to do with a 'common file upload drective'? What is the 'common upload directive' anyway, aren't we adding this in this ticket?

**Comment by pgesek** [comment]
No point in merging this before we fix the uglification issue

File: message-campaign/src/main/resources/webapp/partials/settings.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h4 class="frame-title">{{msg('msgCampaign.settings.title')}}</h4>
         <div class="form-group fileiput fileinput-new" data-provides="fileinput">
             <label class="col-md-2 col-sm-3 control-label">{{msg('msgCampaign.settings.fileChosen')}}</label>
             <div class="col-md-8 col-sm-7 form-inline">
-                <motech-file-upload accept="&#46;json"></motech-file-upload>
+                <motech-file-upload additional="accept:&#46;json"></motech-file-upload>
```
Comment (kmadej) [inline message-campaign/src/main/resources/webapp/partials/settings.html]: revert to previous version without 'additional'

File: message-campaign/src/main/resources/webapp/partials/settings.html [hunk 3]
```diff
@@ -4,22 +4,14 @@ <h4 class="frame-title">{{msg('msgCampaign.settings.title')}}</h4>
...
             <div class="col-md-8 col-sm-7 form-inline">
-                <div class="form-control uneditable-input input-xlarge fileupload-exists"><span class="glyphicon glyphicon-file"></span>&nbsp;<span class="fileupload-preview"></span></div>
-                <span class="btn btn-default btn-file">
-                    <span class="fileupload-new">{{msg('msgCampaign.button.selectFile')}}</span>
-                    <span class="fileupload-exists">{{msg('msgCampaign.button.changeFile')}}</span>
-                    <input type="file" id="messageCampaigns" name="messageCampaigns" class="input-file" accept="&#46;json" />
-                </span>
-                <a href="#" target="_self" class="btn btn-danger fileupload-exists" data-dismiss="fileupload"><i class="fa fa-trash-o"></i>&nbsp;{{msg('msgCampaign.button.remove')}}</a>
-                <span class="form-hint fileupload-new">{{msg('msgCampaign.settings.messageCampaignsHint')}}</span>
+                <motech-file-upload additional="accept:&#46;json"></motech-file-upload>
```
Comment (kmadej) [inline message-campaign/src/main/resources/webapp/partials/settings.html]: revert to previous version without 'additional'

**Comment by matkwiatkowski** [comment]
test this please

**Comment by kmadej** [comment]
Looks good to me. Checked on firefox and chrome
